network/v1/types.go

```diff
@@ -40,6 +56,9 @@ type ClusterNetworkEntry struct {
 	// CIDR defines the total range of a cluster networks address space.
 	CIDR string `json:"CIDR" protobuf:"bytes,1,opt,name=cidr"`
 	// HostSubnetLength is the number of bits of the accompanying CIDR address to allocate to each node. eg, 8 would mean that each node would have a /24 slice of the overlay network for its pods.
+
+	// +kubebuilder:validation:Minimum=2
```

Comment:
Adding a blank line there will cause the doc string to no longer be associated with the following field; you can see this in the swagger diff